src/query/config/src/outer_v0.rs

```diff
@@ -826,6 +821,10 @@ pub struct QueryConfig {
     /// The maximum timeout in milliseconds since the last insert before inserting collected data.
     #[clap(long, default_value = "0")]
     pub async_insert_stale_timeout: u64,
+
+    #[clap(skip)]
+    #[serde(flatten)]
+    pub idm: IDMConfig,
```

Comment:
Please flat this struct directly. I'm afraid we can't load this value from env.

Reply:
done